document/js-api/index.bs

```diff
@@ -873,18 +873,18 @@ enum TableKind {
 
 dictionary TableDescriptor {
   required TableKind element;
-  required IndexValue initial;
-  IndexValue maximum;
-  IndexType index;
+  required AddressValue initial;
+  AddressValue maximum;
+  AddressType addressType;
 };
 
 [LegacyNamespace=WebAssembly, Exposed=*]
 interface Table {
   constructor(TableDescriptor descriptor, optional any value);
-  IndexValue grow(IndexValue delta, optional any value);
-  any get(IndexValue index);
-  undefined set(IndexValue index, optional any value);
-  readonly attribute IndexValue length;
+  AddressValue grow(AddressValue delta, optional any value);
+  any get(AddressValue index);
```

Comment:
Ah, right you are. I'll make a follow-up PR.

Reply:
Well, actually, this is arguably fine since the current spec uses "index" (predating memory64) and it's quite clear in context that this is referring to the index of an element. We can rename if we want, but I'm actually somewhat inclined to keep it as is. I'll open an issue just for any final bikeshedding.